samples/message-error-handling/Core_7/PlatformLauncher/PlatformLauncher.Core.csproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
I don't think this will work or does it? PlatformLauncher needs full framework

Reply:
Explained to Daniel on Slack that package is multi-targeted but just informs it can't do anything via logging on non-Windows platforms for now.